addons/mrp_landed_costs/security/ir.model.access.csv

```diff
@@ -0,0 +1,12 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+"access_stock_landed_cost_mrp","stock.landed.cost mrp_manager","model_stock_landed_cost","mrp.group_mrp_manager",1,1,1,1
+"access_stock_landed_cost_lines_mrp","stock.landed.cost.lines mrp_manager","stock_landed_costs.model_stock_landed_cost_lines","mrp.group_mrp_manager",1,1,1,1
+"access_stock_valuation_adjustment_lines_mrp","stock.valuation.adjustment.lines mrp_manager","stock_landed_costs.model_stock_valuation_adjustment_lines","mrp.group_mrp_manager",1,1,1,1
+
+"access_account_journal_mrp","account.journal mrp_manager","account.model_account_journal","mrp.group_mrp_manager",1,1,0,0
+"access_account_move_mrp","account.move mrp_manager","account.model_account_move","mrp.group_mrp_manager",1,1,1,1
+"access_account_move_line_mrp","account.move.line mrp_manager","account.model_account_move_line","mrp.group_mrp_manager",1,1,1,1
+"access_account_partial_reconcile_mrp","account.partial.reconcile mrp_manager","account.model_account_partial_reconcile","mrp.group_mrp_manager",1,1,1,1
```

Comment:
It's super stange to give that much access to the mrp manager for accouting (rule and access). I'm not sure it's necessary since he can't reach the accounting views without any extra right